packages/backend/src/tokens/types.ts

```diff
@@ -0,0 +1,14 @@
+import type { VerifyTokenOptions } from './verify';
+
+type RequestStateParams = {
+  publishableKey?: string;
+  domain?: string;
+  isSatellite?: boolean;
+  proxyUrl?: string;
+  signInUrl?: string;
+  signUpUrl?: string;
+  afterSignInUrl?: string;
+  afterSignUpUrl?: string;
+};
+
+export type AuthenticateRequestOptions = RequestStateParams & VerifyTokenOptions;
```

Comment:
Can we consolidate these two types, or are we using `RequestStateParams` independently somewhere?

Reply:
Thanks for spotting that, done!